Redirect fixes (#945)
ably/docs: README.md

Then, when accessing `/redirect-from-this-path/`, you'll be redirected to the affected template.

> **IMPORTANT:** The values used in the `redirect_from` list should be the original URL's, not the name of the file that the URL was generated from.

Reviewer: ```suggestion
**IMPORTANT:** The values used in the `redirect_from` list should be the URL without the file extension. E.g. `general/versions/v0.8/webhooks` not `general/versions/v0.8/webhooks.textile`. 
```
